README.md

```diff
@@ -1,3 +1,29 @@
-# Paper Extension
-A chrome extension to sync the highlighted text from web to a dropbox paper document. 
+<h1  align='center'>
+Paper extension
+</h1> 
+<p align='center'>
+<img src='./img/icon-large.png' >
+</p>
+<h5 align='center'>
+Take notes without leaving the tab 
+</h5> 
 
+------------
+
+### Download
+- Download the .zip file of extension using the green coloured *Clone or download*  button.
+- Extract its content at the desired location.
+
+### Authorize
+- Get your *client_id* by following the instructions [here](http:/https://auth0.com/docs/connections/social/dropbox/ "here") and fill in the "*enter client id*" placeholder in paper.js.
+
+### Install
+- Go to [chrome://extensions/](chrome://extensions/ "chrome://extensions/") and toggle the *Developer Mode* ON.
+- Click on load *UNPACKED EXTENSION* and select the folder where you've extracted the extension.
+- The extension is now installed and ready for use.
+
+### Use
+- Click on the paper-extension icon and authorize.
+- The extension is active now. Right-click on the selected text and you'll see the *Save to paper* option!
+
+##### Thank you!
```

Comment:
Rather than a "Thank you" maybe, a short note to other developers on contributing to this project would do! Something like what we've followed in the other projects, "Open issues and PRs, feel free to contribute"